<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>191</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>184</width>
    <height>162</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ArduFoamCutter</string>
  </property>
  <widget class="QLabel" name="selected_file_l">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>370</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>File:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_com_ports_b">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open_port_b">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open port</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_file_b">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send_b">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_port_b">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close port</string>
   </property>
  </widget>
  <widget class="QComboBox" name="com_ports_cb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="status_l">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>370</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="x_axis_b">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>40</width>
     <height>23</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>X+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="y_axis_b">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>40</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Y+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a_axis_b">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>40</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>A+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="z_axis_b">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>40</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Z+</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="step_sp">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="n_x_axis_b">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>40</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>X-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="n_a_axis_b">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>40</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>A-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="n_z_axis_b">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>40</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Z-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="n_y_axis_b">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>40</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
